tests/get_data_test.go

```diff
@@ -0,0 +1,52 @@
+package tests
```

Comment:
We don’t have a separate tests package in the nftables package. 

These tests look different than the ones we already have. I think yours might not work on the GitHub Actions Continuous Integration as they are.

Reply:
> It seems like you’re changing file permissions of all files from 644 to 755 for some reason? Maybe an accidental misconfiguration on your end.

Because the file permission is set to 644, my Goland cannot open the project, so I changed it to 777. I re-decided on a better method, thanks for reminding

Reply:
Regarding the test file, I will confidently study the nftables_test file in the project and add it here. In addition, I hope the README file can provide more cases for users to know how to get started quickly